addons/l10n_es_edi_tbai/data/template_invoice.xml

```diff
@@ -0,0 +1,257 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <data>
+        <template id="template_invoice_main_post">
+            <T:TicketBai xmlns:T="urn:ticketbai:emision">
+                <t t-call="l10n_es_edi_tbai.template_invoice_bundle"/>
+            </T:TicketBai>
+        </template>
+
+        <template id="template_invoice_main_cancel">
+            <T:AnulaTicketBai xmlns:T="urn:ticketbai:anulacion">
+                <t t-call="l10n_es_edi_tbai.template_invoice_bundle"/>
+            </T:AnulaTicketBai>
+        </template>
+
+        <template id="template_invoice_bundle">
+            <Cabecera>
+                <IDVersionTBAI t-out="tbai_version"/>
+            </Cabecera>
+            <Sujetos t-if="is_emission">
+                <t t-call="l10n_es_edi_tbai.template_invoice_sujetos"/>
+            </Sujetos>
+            <Factura t-if="is_emission">
+                <t t-call="l10n_es_edi_tbai.template_invoice_factura"/>
+            </Factura>
+            <IDFactura t-if="not is_emission">
+                <t t-call="l10n_es_edi_tbai.template_invoice_sujetos"/>
+                <t t-call="l10n_es_edi_tbai.template_invoice_factura"/>
+            </IDFactura>
+            <HuellaTBAI>
+                <EncadenamientoFacturaAnterior t-if="chain_prev_invoice">
+                    <SerieFacturaAnterior t-out="chain_prev_invoice.l10n_es_tbai_sequence"/>
+                    <NumFacturaAnterior t-out="chain_prev_invoice.l10n_es_tbai_number"/>
+                    <FechaExpedicionFacturaAnterior t-out="datetime_strftime(chain_prev_invoice.l10n_es_tbai_registration_date, '%d-%m-%Y')"/>
```

Comment:
The only value for which we use specifically the Basque timezone is `datetime_now` for XML tags `FechaExpedicionFactura` and `HoraExpedicionFactura` as those are not to be greater than Basque time (so I assume they have to correspond to Basque time).
There's also the field `l10n_es_tbai_registration_date` where we inject the tzinfo because it is provided by the Basque government and we assume their datetime comes from their timezone.
All the date formatting methods are called on one of the above value/field so I assume there's no need for further conversion.

Reply:
@odoostan Actually, this seems mostly all right.  For everything that is dates itself, we should not manipulate timezones (and for the compute field, just remove that) and the only datetime seems to be the datetime_now where it is done.